app/views/practices/form/metrics.html.erb

```diff
@@ -0,0 +1,302 @@
+<% provide :main_classes, 'bg-gray-2' %>
+<% provide :footer_classes, 'bg-gray-2' %>
+
+<% provide :head_tags do %>
+  <%= javascript_include_tag 'metrics_page', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'practice_editor_utilities', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'https://www.gstatic.com/charts/loader.js' %>
+  <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
+  <% end %>
+<% end %>
+
+<%= render 'practice_editor_sidenav' %>
+<div class="grid-container position-relative">
+  <div class="grid-row grid-gap practice-editor-min-height">
+    <div class="desktop:grid-col-3 grid-col-auto z-bottom">&nbsp;</div>
+
+    <div id="contact" class="desktop:grid-col-9 grid-col-12 p0">
```

Comment:
We should change the id to `metrics` instead of `contact`.

Reply:
done